api/src/passage/management/commands/fix_corrupt_rijrichting.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+import logging
+import time
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.models import Case, Value, When
+
+from iotsignals import settings
+from passage.models import Passage
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('--sleep', nargs='?', default=1, type=float)
+
+    def handle(self, **options):
+        logger.info("message")
+
+        sleep = options['sleep']
+
+        date_min = datetime.datetime(2022, 5, 6, 13, 0)
+        date_max = datetime.datetime.today()
+
+        for date in (
+            date_min + timedelta(n) for n in range((date_max - date_min).days + 1)
+        ):
+            if date != date_min:
+                # for the first date we want a specific timestamp (13:00). For all
+                # following days we want the full day (starting at 00:00)
+                date = date.date()
+
+            self.stdout.write(f"Selecting data in: {self.style.SQL_KEYWORD(date)}")
+            num_updated_rows = Passage.objects.filter(
+                passage_at__gte=date,
+                passage_at__lt=date + timedelta(days=1),
```

Comment:
Begrijp ik dit goed dat we hiermee records tussen 00:00 en 13:00 twee keer gaan processen? Dat kan wel fout gaan omdat de records al een keer omgedraaid zijn.